src/Hazelcast.Net/DistributedObjects/ReliableTopicMessageEventArgs.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Diagnostics.CodeAnalysis;
+using Hazelcast.Models;
+
+namespace Hazelcast.DistributedObjects
+{
+    /// <summary>
+    /// Represents event data for the reliable topic message.
+    /// </summary>
+    /// <typeparam name="T">The reliable topic object type.</typeparam>
+    [ExcludeFromCodeCoverage]
```

Comment:
Why exclude that class?

Reply:
It just contains contains set of getters and setters. I don't see a point to test or decrease the coverage.

Reply:
Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/825/commits/b2aafb0da89c14280eb034330b6b68371c55146d